doc/ipad.md

```diff
@@ -7,7 +7,7 @@
   - [Servediter iPad App](#servediter-ipad-app)
 - [Raspberry Pi USB-C Network](#raspberry-pi-usb-c-network)
 - [Recommendations](#recommendations)
-- [By 2022 iPad coding more desirable on Arm Macs](#by-2022-iPad-coding-more-desirable-on-arm-macs)
+- [By 2022 iPad coding more desirable on Arm Macs](#by-2022-ipad-coding-more-desirable-on-arm-macs)
```

Comment:
I believe this PR needs to be rebased on master.

I fixed this in https://github.com/cdr/code-server/commit/4185675338767c4b73eca7a23e80488bece364e6